src/class/object_genome_builds.php

```diff
@@ -0,0 +1,102 @@
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LOVD.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *************/
+
+// Don't allow direct access.
+if (!defined('ROOT_PATH')) {
+    exit;
+}
+// Require parent class definition.
+require_once ROOT_PATH . 'class/objects.php';
+
+
+
+class LOVD_GenomeBuild extends LOVD_Object
+{
+    // This class extends the basic Object class and it handles the Genome Builds.
+    var $sObject = 'Genome_Build';
+    var $sTable = 'TABLE_GENOME_BUILDS';
+
+
+
+
+
+    function __construct ()
+    {
+        // Default constructor.
+
+        // SQL code for viewing an entry.
+        $this->aSQLViewEntry['SELECT']   = 'gb.*, u.name AS created_by_';
+        $this->aSQLViewEntry['FROM']     = TABLE_GENOME_BUILDS . ' AS gb ' .
+            'LEFT OUTER JOIN ' . TABLE_USERS . ' AS u ON (gb.created_by = u.id) ';
+        $this->aSQLViewEntry['GROUP_BY'] = 'gb.id';
+
+        // SQL code for viewing a list of entries.
+        $this->aSQLViewList['SELECT']   = 'gb.*, u.name AS created_by_';
+        $this->aSQLViewList['FROM']     = TABLE_GENOME_BUILDS . ' AS gb LEFT OUTER JOIN ' . TABLE_USERS . ' AS u ' .
+                                          'ON (gb.created_by = u.id)';
+        $this->aSQLViewList['ORDER_BY'] = 'id';
+
+        // List of columns and (default?) order for viewing an entry.
+        $this->aColumnsViewEntry =
+            array(
+                'TableHeader_General' => 'General information',
+                'id' => 'Genome build ID',
+                'name' => 'Genome build name',
+                'column_suffix' => 'Column suffix',
+                'created_by' => 'Created by',
+                'created_date' => 'Date created'
+            );
+
+        // List of columns and (default?) order for viewing a list of entries.
+        $this->aColumnsViewList =
+                 array(
+                        'id' => array(
+                                    'view' => array('ID', 50),
+                                    'db'   => array('gb.id', 'ASC', true)),
+                        'name' => array(
+                                    'view' => array('name', 50),
+                                    'db'   => array('gb.name', 'ASC', true)),
+                        'created_date' => array(
+                                    'view' => array('created on', 130),
+                                    'db'   => array('gb.created_by', 'DESC', 'DATETIME')),
+                        'created_by_' => array(
+                                    'view' => array('User', 160),
+                                    'db'   => array('u.name', 'ASC', true)),
+                        'column_suffix' => array(
+                                    'view' => array('Column suffix', 100),
+                                    'db'   => array('gb.column_suffix', 'ASC', true))
```

Comment:
Also, please standardize the order of columns. `created_date` goes last, `created_by` before that (basically, use the same order of the fields in the database table).